.gitignore

```diff
@@ -1,5 +1,5 @@
 # Binaries for programs and plugins
-*.exe
+speedtstMirceaD.exe
```

Comment:
I narrowed down to include .exe files in general because I included the SpeedTest.net CLI API redistributable in ./External which is a .exe and is required.